apps/public/src/app/(general)/bedankt/page.test.tsx

```diff
@@ -38,13 +39,15 @@ describe('Page', () => {
   })
 
   it('renders Thanks page', async () => {
-    mockCookies.get.mockReturnValue({ value: '1234' })
+    mockCookies.get.mockReturnValueOnce({ value: '1234' })
+    mockCookies.get.mockReturnValueOnce({ value: '2025-05-26T11:56:34.081Z' })
```

Comment:
We use [this pattern](https://github.com/Amsterdam/meldingen-frontend/blob/main/apps/public/src/app/(general)/bijlage/actions.test.ts#L43) to mock multiple cookies.